apps/cookbook/src/app/showcase/alert-showcase/alert-showcase.component.html

```diff
@@ -37,7 +37,30 @@
     ...
 {{ '}' }}</cookbook-code-viewer>
   </p>
-  <h4>Alert config properties:</h4>
+
+  <h2>Accessibility</h2>
+  <p>
+    Alerts have a
+    <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role">
+      dialog
```

Comment:
There's an accidental space somewhere in here: 
<img width="843" alt="image" src="https://github.com/user-attachments/assets/1f978f2c-52ee-4b7b-b56e-c600ada2bc37">

Reply:
Can't find it, can you help me locate it?